core/client/app/components/gh-posts-list-item.js

```diff
@@ -22,7 +22,7 @@ export default Component.extend(ActiveLinkWrapper, {
     }),
 
     authorAvatar: computed('post.author.image', function () {
-        return this.get('post.author.image') || this.get('ghostPaths.url').asset('/shared/img/user-image.png');
+        return this.get('post.author.image') || `${this.get('ghostPaths.subdir')}/ghost/img/user-image.png`;
```

Comment:
Won't that require authentication and hence break in themes that rely on this picture? (or is it up to the theme to decide what to do if the image is not existent?)